### ISSUE TITLE
[@typescript-eslint/indent] False positives in enums

### ISSUE THREAD

**Issue by manuth** [typescript-eslint/typescript-eslint#1701]
**Repro**
```json
{
  "rules": {
    "@typescript-eslint/indent": "warn"
  }
}
```

```ts
export enum LogLevel {
    Info,
    Warning
}

export enum LogLevel
{ // Expected indentation of 4 spaces but found 0. (@typescript-eslint/indent)
    Info,
    Warning
}
```

**Expected Result**
Both examples should work as expected.

**Actual Result**
An error is reported in the second example.

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `2.22.1-alpha.6` |
| `@typescript-eslint/parser`        | `2.22.0` |
| `TypeScript`                       | `3.8.3` |
| `ESLint`                           | `6.8.0` |
| `node`                             | `v12.4.0` |
| `npm`                              | `6.9.0` |

**Comment by bradzacher** [comment]
recommendation for this an #1700 is to not use the rule.
It's very broken, and unlikely to get fixed due to the huge complexity in the rule.
You should consider using a tool like prettier instead.

**Comment by bradzacher** [comment]
Merging into #1824
